app/main/forms/direct_award_forms.py

```diff
@@ -14,3 +15,14 @@ class CreateProjectForm(FlaskForm):
             Optional()
         ]
     )
+
+
+class WhyDidntYouAwardForm(FlaskForm):
+    why_didnt_you_award_contract = RadioField(
```

Comment:
So if we must have URL's that are missing punctuation, then that's a Content thing, and I can live with it. But please can we not have functions or classes named that way!